cpp/Ice/config/Chatbot.cpp

```diff
@@ -6,10 +6,9 @@
 #include <sstream>
 
 using namespace std;
-using namespace ConfigServer;
```

Comment:
Also removed `using namespace` to make it clearer what comes from the Server namespace.